guardrails/cli/configure.py

```diff
@@ -1,17 +1,25 @@
 import os
-import sys
 import uuid
 from os.path import expanduser
 from typing import Optional
 
+from guardrails.cli.server.hub_client import get_auth
 import typer
 
 from guardrails.cli.guardrails import guardrails
 from guardrails.cli.logger import LEVELS, logger
-from guardrails.cli.server.hub_client import AuthenticationError, get_auth
 
 
-def save_configuration_file(token: str, no_metrics: bool) -> None:
+DEFAULT_TOKEN = ""
+DEFAULT_NO_METRICS = False
+
+
+def save_configuration_file(token: Optional[str], no_metrics: Optional[bool]) -> None:
+    if token is None:
+        token = DEFAULT_TOKEN
+    if no_metrics is None:
+        no_metrics = DEFAULT_NO_METRICS
```

Comment:
Not for this task, but we should really update the nomenclature here.  The double negatives are a usability issue and, in this instance, kind of a transparency issue too.  We should just set it to an affirmative flag `allow_metrics` or something like that.

Reply:
For this task!